src/components/Group/GroupMemberCard.tsx

```diff
@@ -0,0 +1,146 @@
+import { Role } from "@prisma/client";
+import { PublicUser } from "../../utils/types";
+import { useContext } from "react";
+import { UserContext } from "../../utils/userContext";
+import Spinner from "../Spinner";
+import _ from "lodash";
+import { trpc } from "../../utils/trpc";
+import { toast } from "react-toastify";
+import { TRPCClientError } from "@trpc/client";
+import { useToasts } from "react-toast-notifications";
+
+interface GroupMembersProps {
+  users: PublicUser[];
+  onClose: () => void;
+}
+export const GroupMembers = (props: GroupMembersProps) => {
```

Comment:
Just a minor comment but I think declaring the interface type and having props as the component input is good style and has been consistent with how we've been designing this app for a while, but it's different from how I remember some of our original components and pages were styled where the input was a restructured object with a custom type declared in the function input. I saw CustomPopUp.tsx uses a similar approach to this (although it uses a predefined type as input), but there might be older components that still have this. Maybe should we consider making a small ticket to just look for these (if any still remain) and update them to all use this props input style?